src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By CustomPipeName.Can enter, exit, and re-enter using CustomPipeName
```powershell
Expected $true, because The script was able to re-enter another process and grab the pipe of '4qxs3kou.5m3'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 170
170:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pipe of '$pipeName'."
```